src/TSMapEditor/Mutations/Classes/PlaceOverlayMutation.cs

```diff
@@ -29,9 +29,13 @@ public PlaceOverlayMutation(IMutationTarget mutationTarget, OverlayType overlayT
         public override string GetDisplayString()
         {
             if (overlayType != null)
-                return $"Place overlay '{overlayType.GetEditorDisplayName()}' at {cellCoords} with a brush size of {brush}";
+                return string.Format(Translate(this, "DisplayStringPlace", 
+                    "Place overlay '{0}' at {1} with a brush size of {2}"),
+                        overlayType.GetEditorDisplayName(), cellCoords, brush);
             else
-                return $"Erase overlay at {cellCoords} with a brush size of {brush}";
+                return string.Format(Translate(this, "DisplayStringErase", 
+                    "Erase overlay at {0} with a brush size of {1}"),
+                        cellCoords, brush);
```

Comment:
From the contribution guideline:

> Braceless code block bodies should be made only when both code block head and body are single line. Statements that split into multiple lines ... are not allowed within braceless blocks

```suggestion
            {
                return string.Format(Translate(this, "DisplayStringErase", 
                    "Erase overlay at {0} with a brush size of {1}"),
                        cellCoords, brush);
            }
```